### PR TITLE
Added polling to the status to see any changes as they happen - this …

### PR DESCRIPTION
**Pull request by aaron-russell**
…wont work for upload

### CODE REVIEW

**Comment by aaron-russell** [comment]
[(3) Trigger Status API on BU Page Load](https://trello.com/c/CyblJ8Cg/26-3-trigger-status-api-on-bu-page-load)

File: src/app/features/bulk-upload/files-upload/files-upload.component.ts [hunk 1]
```diff
@@ -43,7 +45,16 @@ export class FilesUploadComponent implements OnInit, AfterViewInit {
   ngOnInit() {
     this.setupForm();
     this.checkForUploadedFiles();
-    // this.checkBulkUploadState();
+    this.status = interval(1000).subscribe(() => {
+      console.log(this.stopPolling);
```
Comment (Negi-Anurag) [inline src/app/features/bulk-upload/files-upload/files-upload.component.ts]: Remove console.log()

File: src/app/features/bulk-upload/files-upload/files-upload.component.ts [hunk 2]
```diff
@@ -43,7 +45,16 @@ export class FilesUploadComponent implements OnInit, AfterViewInit {
   ngOnInit() {
     this.setupForm();
     this.checkForUploadedFiles();
-    // this.checkBulkUploadState();
+    this.status = interval(1000).subscribe(() => {
```
Comment (Negi-Anurag) [inline src/app/features/bulk-upload/files-upload/files-upload.component.ts]: Do we need to cancel this interval onDestroy , as I can't see you haven't stopped this interval.

File: src/app/features/bulk-upload/files-upload/files-upload.component.ts [hunk 3]
```diff
@@ -43,7 +45,16 @@ export class FilesUploadComponent implements OnInit, AfterViewInit {
...
+      console.log(this.stopPolling);
+      if (!this.stopPolling) {
+        this.checkForUploadedFiles();
+      }
+    });
+  }
+
+  ngOnDestroy() {
+    this.status.unsubscribe();
```
Comment (aaron-russell) [inline src/app/features/bulk-upload/files-upload/files-upload.component.ts]: It cancels here @Negi-Anurag

Comment (Negi-Anurag) [reply src/app/features/bulk-upload/files-upload/files-upload.component.ts]: Got it.

**Pull request merged**